tests/data/mock_manifests/Invalid_Test_Manifest.csv

```diff
@@ -1,4 +1,4 @@
 Component,Check List,Check Regex List,Check Regex Format,Check Regex Single,Check Num,Check Float,Check Int,Check String,Check URL,Check Match at Least,Check Match at Least values,Check Match Exactly,Check Match Exactly values,Check Recommended,Check Ages,Check Unique,Check Range,Check Date
-MockComponent,"ab,cd","ab,cd,ef",a,a,6,99.65,7,valid,https://www.google.com/,1738,1738,8085,98085,,6549,str1,70,32984
+MockComponent,"ab,cd","ab,cd,ef",a,a,6,99.65,7,valid,https://www.google.com/,1738,1738,8085,98085,,6549,str1,70,32-984
```

Comment:
Can you add assertions in `test_validation.TestManifestValidation.test_invalid_manifest` that confirm the appropriate error messages are generated?